pkg/combustion/cleanup.go

```diff
@@ -0,0 +1,37 @@
+package combustion
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/fileio"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+	"github.com/suse-edge/edge-image-builder/pkg/log"
+	"go.uber.org/zap"
+)
+
+const (
+	cleanupScriptName    = "99-remove-combustion.sh"
```

Comment:
Nit: I'd suggest `cleanup-combustion` instead.

Reply:
It's funny, I force pushed this change right before you commented this because I saw it and thought the same thing

Reply:
I wonder if we want to use something like `zz-cleanup-combustion.sh` instead of numbers. We currently don't force users into our numbering prefix scheme, but `99-` will run before any non-number-prefixed script, so anyone with custom scripts is gonna have a bad time when combustion is deleted before their script runs.

Reply:
Actually, we may want to treat this entirely outside of the alphabetical scheme and instead of using the list of scripts to reference from the Combustion `script` file (worst name ever), we just append this to the end of that list regardless.